### ISSUE TITLE
Is Paper the only supported server?

### ISSUE THREAD

**Issue by Dartsgame974** [flokol120/Spigot-Item-Chest-Sorter#40]
Hello

the plugin works on another api than Paper?

**Comment by flokol120** [comment]
Hi @Dartsgame974, I am sorry, but I don't understand your question. Could you rephrase it?

**Comment by Dartsgame974** [comment]
The plugin work only with paper or I can use anthoer version like spigot or mohist?

**Comment by flokol120** [comment]
@ I see. The plugin was originally intended for Spigot. Paper support is new. I don't know what Mohist, hence it's not tested.

**Comment by flokol120** [comment]
@Dartsgame974 What version of the plugin and paper are you using? Is there any error logged?

**Comment by Dartsgame974** [comment]
i don't use paper i use purpur 
![image](https://user-images.githubusercontent.com/80755313/170372915-6aabb73a-1f09-4a51-87cf-a44171a36b2c.png)

**Comment by flokol120** [comment]
@Dartsgame974 I haven't heard of Purpur either (I thought you meant paper and made a typo :D). The Purpur FAQ states this (https://purpurmc.org/docs/FAQ/#do-craftbukkitspigotpaperpufferfish-plugins-work-on-purpur)

_Yes. The only time there's incompatibility is due to authors hard-coding support for CraftBukkit/Spigot, ignoring the existence of Paper and its forks. If you run into any bugs that you know are because of Purpur, [create a ticket](https://github.com/PurpurMC/Purpur/issues/new) and/or [join our Discord server](https://purpurmc.org/discord) so we can take a look._

If I hardcoded anything, I am not aware of. Either way there are no instructions on how to resolve or identify the issue. The documentation is also not helping...

So long story short: If it does not work: Purpur is not supported. I don't know how to make it run on Purpur

If there happen to be more reports of people trying to use Purpur, I'll take a deeper look into this direction. Closed for now. If you happen to have any issues on Spigot or Paper, feel free let me know.
